compose/backend.yml

```diff
@@ -23,6 +23,8 @@ services:
     links:
       - ccd-shared-database
       - service-auth-provider-api
+    networks:
+      - ccd-network
```

Comment:
moved from default docker network bridge compose_default to custom external one ccd-network (please run ./ccd init to create the network). This is needed to allow seamless switching between Idam and Idam Stub